lms/djangoapps/bulk_email/tests/test_course_optout.py

```diff
@@ -114,3 +119,56 @@ def test_optin_course(self):
         sent_addresses = [message.to[0] for message in mail.outbox]
         self.assertIn(self.student.email, sent_addresses)
         self.assertIn(self.instructor.email, sent_addresses)
+
+    def test_policy_optedout(self):
```

Comment:
Nit: For readability and self-documenting code, consider putting these test methods in a separate Test class for ACE-specific policy tests.